src/test/java/mat/shared/PasswordVerifierTest.java

```diff
@@ -1,40 +1,67 @@
 package mat.shared;
 
-import junit.framework.TestCase;
+import org.junit.Test;

```

Comment:
interesting...
You rely on Jnit5 assertions, but JUnit5 '@Test' annotation.
I would suggest making it consistent and use annotation from org.junit.jupiter.api
Not critical.

Reply:
Good catch will change that.

Reply:
Good catch. Will change it to use Assert.